relay-spans/Cargo.toml

```diff
@@ -13,8 +13,10 @@ publish = false
 chrono.workspace = true
 enumset = "1.0.4"
 once_cell.workspace = true
+opentelemetry-proto = { git = "https://github.com/open-telemetry/opentelemetry-rust", features = ["gen-tonic", "with-serde", "trace"] }
```

Comment:
Not a fan of a git dependency but if we have to it should always include a fixed commit hash (use `rev = `)!